DashAI/back/dataloaders/classes/csv_dataloader.py

```diff
@@ -71,13 +72,14 @@ def load_data(
             dataset = load_dataset("csv", data_files=url, sep=separator)
         elif file:
             files_path = self.extract_files(dataset_path, file)
-            try:
-                if files_path.split("/")[-1] == "files":
+            if files_path.split("/")[-1] == "files":
```

Comment:
What happens if this operation fails?

Reply:
If there is an error in the extract files that will throw the error or do you mean something else?

Reply:
Yes, that thing.
I was telling that the back will raise a strange exception, with no clue of what really failed.
But, I guess that we could fix the error handling in future work :)